docs/section11/Rule11-15.md

```diff
@@ -0,0 +1,67 @@
+
+    ## Rule Logic:
+    - get the proposed swh use: `swh_use_p = get_component_by_id(swh_use_id, P_RMD)`
+    - get the proposed swh use: `swh_use_b = get_component_by_id(swh_use_id, B_RMD)`
+    - create a value to track the rule status: `rule_status = "PASS"`
+    - create a variable to track any notes: `rule_note = ""`
+    - if swh_use_b is null: `if swh_use_b == nil:`
+        - set the rule status to FAIL and provide note: `rule_status = "FAIL"; note = swh_use_id + "exists in the Proposed model, but not in the baseline.`
+        - continue to rule assertion: `CONTINUE TO RULE ASSERTION`
+    - if swh_use_P is null: `if swh_use_P == nil:`
+        - set the rule status to FAIL and provide note: `rule_status = "FAIL"; note = swh_use_id + "exists in the Baseline model, but not in the proposed.`
+        - continue to rule assertion: `CONTINUE TO RULE ASSERTION`
+    - check that the use matches by checking that the use, use_units, use_multiplier_schedule, and temperature_at_fixture all match (ARE THERE OTHERS THAT NEED TO MATCH TOO?)
+    - check use_units, if not equal, set to fail and add a note indicating that the use units are different: `if swh_use_p.use_units != swh_use_b.use_units: rule_status = "FAIL"; rule_note += "Service water heating use units are inconsistent between proposed and baseline models.  "`
+    - check use_multiplier_schedule, if not equal set to fail and append a note indicating that the use_multiplier_schedules don't match: `if swh_use_p.use_multiplier_schedule != swh_use_b.use_multiplier_schedule: rule_status = "FAIL"; rule_note += " Service Water Heating Use schedules do not match.  "`
+    - check temperature_at_fixture, if not equal set to fail and append a note indicating that the temperature_at_fixture doesn't match: `if swh_use_p.temperature_at_fixture != swh_use_b.temperature_at_fixture: rule_status = "FAIL"; rule_note += " The temperature at fixture is not the same between Proposed and Baseline.`
+    - now, if the rule_status is still "PASS", we'll check if the use is equal or not between proposed and baseline: `if rule_status == "PASS":`
+      - check whether the proposed use is less than the baseline, this will result in UNDETERMINED as an ECM could reduce swh use due to low-flow fixtures.  Change rule_status and append note: `if swh_use_p.use < swh_use_b.use: rule_status = "UNDETERMINED"; rule_note = "Proposed Service Water Heating Use is less than the baseline.  Manually verify that reduction is due to an ECM that reduces service water heating use, such as low-flow fixtures.  "`
+      - otherwise, if the proposed use is greater than the baseline, the result is fail.  Provide a note: `if swh_use_p.use > swh_use_b.use: rule_status = "FAIL"; rule_note += "Proposed Service Water Heating Use is greater than the baseline.  "`
+
+     **Rule Assertion - Zone:**
+    - Case1: rule_status is PASS: `if rule_status == "PASS": PASS`.
```

Comment:
I think this rule also needs to address cpmpliance with 7.4.1. Perhaps it shoudl be two separate rules. One that checks baselein and proposed and nother that jsut checks if there are loads and then directs the reviewer toensure compliance with 7.4.1 calculations procedures
.

Reply:
11-9 already states that the baseline needs to be sized according to 7.4.1.  So perhaps we can remove the language about 7.4.1 from this rule?

Reply:
@KarenWGard I agree with that approach.